src/arrow-right.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="29" height="19" viewBox="0 0 29 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please clean up assets in the project. You can for example move them to `assets` directory and divide into `avatars` and `icons` directory.